circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
     version: "9.1"
-dependencies:
-  cache_directories:
-    - "Carthage"
+test:
+  override:
+   - set -o pipefail && xcodebuild clean build test -project Daylight.xcodeproj -scheme Daylight -destination 'platform=iOS Simulator,name=iPhone 7' | xcpretty --color && exit ${PIPESTATUS[0]}
```

Comment:
You may want to update this to the Circle 2.0 format since they're deprecating 1.0 and not updating it past Xcode 9.1. Again: Not necessary for this PR, just recommended.